frontend/src/App.tsx

```diff
@@ -16,6 +17,10 @@ function App() {
   useEffect(() => {
     // eslint-disable-next-line prettier/prettier
     (async () => {
+
+      ReactGA.initialize('G-R730Q5P12D')
```

Comment:
this seems to be a key that should be hidden in env

Reply:
What are you referring to? The key is needed in browser, I don't think we can hide it in an env.